ada_feeding/config/ada_planning_scene.yaml

```diff
@@ -55,4 +60,12 @@ ada_planning_scene:
       primitive_dims: [1.59, 1.5, 0.01] # Box has 3 dims: [x, y, z]
       position: [-0.05, 0.17, 1.05]
       quat_xyzw: [0.0, 0.0, 0.0, 1.0]
-      frame_id: root # the frame_id that the pose is relative to
\ No newline at end of file
+      frame_id: root # the frame_id that the pose is relative to
```

Comment:
Just making sure that root is pre-tilt, in case the wheelchair is tilted.

Reply:
Yup, `root` is pre-tilt, `root_tilt` is post-tilt